src/pages/Admin/Charity/EditProfile/Form.tsx

```diff
@@ -43,13 +42,12 @@ export default function Form() {
       onSubmit={editProfile}
       className="w-full max-w-4xl justify-self-center grid content-start gap-6 mt-6"
     >
-      <Link
-        to={`${appRoutes.marketplace}/${id}`}
+      <ExtLink
+        href={`${appRoutes.marketplace}/${id}`}
         className="text-blue-d1 hover:text-navy text-sm flex items-center gap-1"
       >
-        <Icon type="Back" />
-        <span>Back to profile</span>
-      </Link>
+        View Profile
```

Comment:
Related to https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2842#issuecomment-2036297749:
> @SovereignAndrey, new-window link looks isolated, perhaps we could move this right next to publish profile?

Maybe even move it a bit down, to be inline with "Publish Profile"? Something like:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/d6d53cfa-a851-4bb7-a242-bfcec2deb1fc)

Reply:
> Related to [#2842 (comment)](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2842#issuecomment-2036297749):
> 
> > @SovereignAndrey, new-window link looks isolated, perhaps we could move this right next to publish profile?
> 
> Maybe even move it a bit down, to be in line with "Publish Profile"? Something like: ![image](https://private-user-images.githubusercontent.com/19427053/319503753-d6d53cfa-a851-4bb7-a242-bfcec2deb1fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTIyMjUyMzIsIm5iZiI6MTcxMjIyNDkzMiwicGF0aCI6Ii8xOTQyNzA1My8zMTk1MDM3NTMtZDZkNTNjZmEtYTg1MS00YmI3LWEyNDItYmZjZWMyZGViMWZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA0VDEwMDIxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzZjA4YjU2YWMzN2Y0MWJjYmM2Y2RhMDk2NzI2MWVkYzEyYTNiNmVlYjdhOGE3YjYwNGNiMGU0OWI1Nzc5NzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.S3XKR0YQPxX6s3jiWCuFLMePknxMsoIITljoTDFUIAw)

Agreed! @ap-justin Let's move that either:
- inline with the publish, as @misicnenad suggested OR
- directly underneath or above  the publish profile toggle